Feature/entries list

subinasr opened the pull request:
- Addresses #2780 
- Depends on [deep-server](https://github.com/the-deep/server/pull/1388)

## Changes

* Fetch draft entries list

## This PR doesn't introduce any:

- [ ] temporary files, auto-generated files or secret keys
- [ ] build works
- [ ] eslint issues
- [ ] typescript issues
- [ ] codegen errors
- [ ] `console.log` meant for debugging
- [ ] typos
- [ ] unwanted comments
- [ ] conflict markers

## This PR contains valid:

- [ ] permission checks
- [ ] translations

subinasr committed:
Add auto extracted draft entries

ghost commented:
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=2794&src=https%3A%2F%2Fgithub.com%2Fthe-deep%2Fclient">Click on the image</a> for a new way to code review</b>
</summary>

####

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/the-deep/client/2794/aa166e63/0cf70d9958841d3ddd1e1f88dc4e43713eb24b6a.svg)](https://app.codesee.io/r/reviews?pr=2794&src=https%3A%2F%2Fgithub.com%2Fthe-deep%2Fclient)

#### Legend
<img src="https://app.codesee.io/Legend_v3.png" alt="CodeSee Map legend" width="220" height="156">
</details>

subinasr committed:
Data transformation for draft entries

AdityaKhatri committed:
Rearrange queries and mutation for auto entries modal

AdityaKhatri committed:
Add ability to add entries from recommendations

subinasr committed:
Make entry input similar to assisted tagging

subinasr committed:
Enable discard/undiscard entry

- Disallow duplicate entries creation.
- Remove selection of entry after creation from auto draft entries

AdityaKhatri committed:
Fix empty message issues in auto extract popup
